Makefile

```diff
@@ -80,3 +80,7 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
+
+roll:
```

Comment:
What's this?

K8s context `aliqa ` is always existing?

Reply:
this context is my local cluster, I will remove this line.